config/initializers/application_controller_renderer.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
```

Comment:
The next few files have all been added as part of the Rails upgrade. They don't appear to do anything for now, but do help document rails features.